gems/pending/blackbox/VmBlackBox.rb

```diff
@@ -88,7 +88,7 @@ def deleteLocalDataDir(options={})
     end
 
     def writeData(filename, data)
-      Dir.mkdir(@localDataDir, 755) unless File.exist?(@localDataDir)

```

Comment:
@simon3z Did you actually run into this or was it something you saw?  I was under the impression this code wasn't in use but not yet able to be deleted due to some dependencies in the miq verbs area (I believe).